projects/api/src/routes/auth.ts

```diff
@@ -0,0 +1,21 @@
+import expressPromiseRouter from 'express-promise-router'
+import {Request, Response, Router} from 'express'
+import {authController} from '../controllers'
+import {validateBody} from '../middlewares'
+import {user as userSchema} from '../schemas'
+import {User} from '../entities/User'
+
+export const authRouter: typeof Router = () => {
```

Comment:
Why ```typeof Router``` ?

Reply:
Because Router is a function, we can write ```const authRouter: Function``` but with typeof Router we know that we are in a routing context